Drivers/rc_receiver/Inc/rcreceiver_datatypes.h

```diff
@@ -37,6 +37,24 @@ struct RCControl{
 
     float &operator[] (int i) { return ControlSignals[i]; }
 
+    void assignValues(RCControl const  &original){
```

Comment:
I don't really understand why this function is needed. I would think to either use the copy constructor, or overload the default equality operator and copy the contents of the `ControlSignals` buffer instead of the value of each element individually.

Reply:
Yup, that is a good idea. I'll try to implement that 👍